### PR TITLE
FISH-8142 Update Mojarra

### PR DESCRIPTION
**Pull request by Viii3**
## Description

- Resolves FISH-8142

### Blockers

- Depends on [Faces TCK 4.0.3](https://github.com/payara/jakartaee-10-tck-runners/pull/74)
- Depends on [Jenkins using the 4.0.3 folder](https://github.com/payara/EngineeringJenkinsjobs/pull/124)

### Testing Performed

Built the server and ran it against Faces TCK 4.0.3

### Testing Environment

Maven version: 3.6.3
Java version: 11.0.23, vendor: Eclipse Adoptium
Default locale: en_GB, platform encoding: UTF-8
OS name: "linux", version: "6.8.0-40-generic", arch: "amd64", family: "unix"

### CODE REVIEW

File: appserver/web/jsf-connector/src/main/java/org/glassfish/faces/integration/GlassFishInjectionProvider.java [hunk 1]
```diff
@@ -473,8 +474,8 @@ public void enableHighAvailability(ServletContext ctx) {
                 }
                 
                 if (isDistributable && enableHA) {
-                    LOGGER.fine("setting the EnableAgressiveSessionDirtying to true");
-                    config.overrideContextInitParameter(EnableAgressiveSessionDirtying, TRUE);
+                    LOGGER.fine("setting the EnableDistributable to true");
```
Comment (Pandrex247) [inline appserver/web/jsf-connector/src/main/java/org/glassfish/faces/integration/GlassFishInjectionProvider.java:477]: Might as well fix the grammar
```suggestion
                    LOGGER.fine("setting EnableDistributable to true");
```

**Review by Pandrex247 — COMMENTED** [review]
Kicked off a run here: https://jenkins.payara.fish/blue/organizations/jenkins/JakartaEE-10-TCK/detail/JakartaEE-10-TCK/253/pipeline/

**Comment by Viii3** [comment]
> Kicked off a run here: https://jenkins.payara.fish/blue/organizations/jenkins/JakartaEE-10-TCK/detail/JakartaEE-10-TCK/253/pipeline/

The TCK succeeded but the run ultimately failed during the summary. Jenkins was looking for the non-existent 4.0.1 folder for the results. I've made a [new PR](https://github.com/payara/EngineeringJenkinsjobs/pull/124) to resolve this.

**Review by breakponchito — APPROVED** [review]
LGTM

**Pull request merged**